packages/vuetify/src/components/VCalendar/mixins/calendar-with-events.sass

```diff
@@ -0,0 +1,52 @@
+@import '../../../styles/styles.sass'
+@import '../_variables.scss'
+
++theme(v-calendar-events) using ($material)
+  .v-event-timed
+    border: 1px solid !important
+
+.v-calendar
+  .v-event
+    position: relative
+    overflow: hidden
+    text-overflow: ellipsis
+    white-space: nowrap
+    font-size: $calendar-event-font-size
+    cursor: pointer
+    margin-right: -1px
+    &.v-event-start
```

Comment:
Please format this and :point_down: with blank lines for clarity like in other styles.